Resources/views/CRUD/edit_orm_many_association_script.html.twig

```diff
@@ -34,6 +34,10 @@ This code manage the many-to-[one|many] association field popup
         initialize_popup_{{ id }}();
 
         var target = jQuery(this);
+        
+        if (target.attr('href').length === 0 || target.attr('href') === '#') {
```

Comment:
That's kind of you ! However my answer was partly wrong...
In this particular context, we're checking the url for a link that has been loaded from a different url.
As the location object always refers to the "top level" location, the first solution won't work. If the loaded html fragment contains a link to the current "top level" page, it is valid to open it in a new dialog.
So the second method is actually the only valid option.